addon/components/file-dropzone/component.js

```diff
@@ -118,16 +118,30 @@ export default Ember.Component.extend({
   didDrop({ originalEvent: evt }) {
     // Testing support for dragging and dropping images
     // from other browser windows
-    let url = evt.dataTransfer.getData('text/uri-list');
-    let html = evt.dataTransfer.getData('text/html');
-    if (html) {
+    let supportsUrls, supportsHtml, html, url;
+
+    try {
+      html = evt.dataTransfer.getData('text/html');
+      supportsHtml = true;
```

Comment:
Would that even be possible if you need an event object to begin with? I'm learning a lot here, so I'd appreciate any insight that you have.